### DISCUSSION TITLE
Cannot pick a specific date span (like April 3 -> April 9)

### DISCUSSION THREAD

**Discussion by sptv2reg** [plausible/analytics#2850] [Anything Else]
I want to pick from 3th - 9th april, but it won't let me do that it seems :-)

https://user-images.githubusercontent.com/103186488/232041833-a172c534-5d56-4472-b290-fbd9565f7b4e.mp4

**Comment by metmarkosaric** [comment]
thanks for reporting @sptv2reg and sorry for the trouble! yeah, it's a new bug introduced few days ago. it shifts the time range to the day before the day you selected. [the fix](https://github.com/plausible/analytics/pull/2848) is already ready for review and deployment. should go live early next week. thanks for your patience!

**Comment by metmarkosaric** [comment]
hi @sptv2reg! just a quick note that this has now been fixed. sorry for the delay!
